graylog2-shared/src/main/java/org/graylog2/shared/security/ldap/LdapEntry.java

```diff
@@ -17,13 +17,18 @@
 package org.graylog2.shared.security.ldap;
 
 import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
 
+import java.util.Collection;
 import java.util.Map;
+import java.util.Set;
 
 public class LdapEntry {
```

Comment:
Is this class ever used in the context of a (Hash)Map or a (Hash)Set? If yes, we need to implement `hashcode()` and `equals()`.